Cake.Recipe/Content/testing.cake

```diff
@@ -266,4 +205,50 @@ BuildParameters.Tasks.IntegrationTestTask = Task("Run-Integration-Tests")
                 });
     });
 
+BuildParameters.Tasks.GenerateFriendlyTestReportTask = Task("Generate-FriendlyTestReport")
+    .IsDependentOn("Test-VSTest")
```

Comment:
Is there a reason that you haven't added some of the other test frameworks here?

Reply:
Since they weren't used on the unit tests already implemented, I assumed ReportUnit did not work for them.
As such, I didn't bother adding those to the dependency list here.